subsys/net/ip/net_context.c

```diff
@@ -1582,6 +1629,7 @@ int net_context_send(struct net_context *context,
 
 int net_context_sendmsg(struct net_context *context,
 			const struct msghdr *msghdr,
+			size_t len,
```

Comment:
same here

Reply:
Actually I removed this one, the len option is not needed.